### ISSUE TITLE
Requirement:The need to adjust the font

### ISSUE THREAD

**Issue by leaf2006** [randy3k/Terminus#288]
Can I add font adjustment in terminal? When I use this plug-in to open the terminal, because some problems of my theme need to be solved by setting the font, the following problems will appear:

![image](https://user-images.githubusercontent.com/51502183/120925967-85e7df00-c70d-11eb-917a-ce4beeb4ce8f.png)

I hope this demand can be solved. Thank you.🙏

**Comment by randy3k** [comment]
Yes, you could specify the font in the preference via `view_settings`.
